__mocks__/MockCollection.ts

```diff
@@ -1,8 +1,10 @@
 export default class MockCollection {
+  findOne;
   deleteOne;
   updateOne;
 
   constructor() {
+    this.findOne = jest.fn(() => Promise.resolve());
```

Comment:
Do we want to actually return a value from `.findOne` for future testing?

Reply:
The pattern that we're following for existing unit tests in this repo is just to override the existing value of these stubbed methods in the individual tests themselves if we want a different return value